robot/robot.go

```diff
@@ -90,9 +90,9 @@ type Robot interface {
 type LocalRobot interface {
 	Robot
 
-	// Config returns the local config used to construct the robot.
-	// This is allowed to be partial or empty.
-	Config(ctx context.Context) (*config.Config, error)
+	// Config returns a config representing the current state of the resource
+	// manager.
```

Comment:
resource manager is a bit low level for the average API user. Can we say current state of the robot?